staging/src/k8s.io/client-go/kubernetes/typed/core/v1/node_expansion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors.
+Copyright 2021 The Kubernetes Authors.
```

Comment:
Hmm, I am not sure if we should be changing the date here.

Reply:
agreed, this should probably not be changed